lib/activerecordbase.rb

```diff
@@ -69,7 +69,8 @@ def queue_update(attributes_of_interest)
                                   attr: k, new_value: v.to_s)
     end
   rescue Exception => e
-    BqStream.log.info "#{Time.now}: EXCEPTION: #{e}"
+    Logger.new("#{File.expand_path('')}/log/bq_stream_queue_update_errors.log",
+               File::WRONLY | File::APPEND).info "#{Time.now}: EXCEPTION: #{e}"
```

Comment:
Does this need to be a different log file?

Reply:
This is the only rescue on an exception. The other log pulls the environment variable from rails. The gem doesn't have access to Rails, especially if ever in a different type of application.

Reply:
Right, so why should we assume we are in a Rails like app structure where the logs go into a directory name `/log`? I think there are a couple of better options:
1. We create a separate logger for just errors that you can enable/disable separately
```ruby
BqStream.error_logger = Logger.new(...)
```
2. We just use the main logger but with a log level set:
```ruby
BqStream.logger = Logger.new(..., level: :error)
```
Which will ignore the info level logs and only put out the errors